### PR TITLE
treewide: Make it compile against Tango 10.0.0

### PR DESCRIPTION
**Pull request by t-b**
The CMakeLists.txt is taken from TangoTest and adapted. And we also added std for all unqualified accesses to the std namespace. In addition we also changed the include path (change from 9.4.0).

### CODE REVIEW

**Comment by bourtemb** [comment]
An update of the Build section of the README.md file would be welcome too.

**Comment by t-b** [comment]
Good point. Done.

File: CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,49 @@
+cmake_minimum_required(VERSION 3.18...3.26 FATAL_ERROR)
+
+project(DynaTest LANGUAGES CXX
+        VERSION 3.0)
+
+list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/TangoCMakeModules)
```
Comment (bourtemb) [inline CMakeLists.txt]: The TangoCMakeModules git submodule seems to be missing at first sight too.

Comment (t-b) [reply CMakeLists.txt]: Well for cppTango 10.0.0 we don't need the tango cmake modules anymore. Should I add them for older versions though?

Comment (bourtemb) [reply CMakeLists.txt]: This project was created to help reproducing https://gitlab.com/tango-controls/cppTango/-/issues/814 .
If we don't need to do the investigation for these versions, no need to add them, then.

Comment (t-b) [reply CMakeLists.txt]: Okay, I've removed that line.

**Pull request merged**